### PR TITLE
Feature/inital schema

### PR DESCRIPTION
**Pull request by jenniferboedker**

### CODE REVIEW

File: model.schema.json [hunk 1]
```diff
@@ -6,9 +6,61 @@
...
+            "pattern": "Q[A-X0-9]{4}ENTITY-[1-9][0-9]*"
+        },
+        "species": {
+            "description": "Scientific name of organism(s) from which the biological material was derived.",
+            "type": "string"
+        },
+        "extractCode": {
+            "description": "QBiC internal identifier for extract samples",
+            "type": "string",
+            "pattern": "Q[A-X0-9]{4}E[1-9][0-9]*"
```
Comment (sven1103) [inline model.schema.json]: This is an experiment code, but should it not be a sample code?

File: validate.groovy [hunk 2]
```diff
@@ -19,9 +19,105 @@ class ValidateSpecification extends Specification {
         schema = SchemaLoader.load(rawSchema)
     }
 
-    def 'this should throw an exception'() {
+    def 'qbicID exception'() {
```
Comment (sven1103) [inline validate.groovy]:
```suggestion
    def 'a malformatted sample id  should throw a validation exception'() {
```

File: validate.groovy [hunk 3]
```diff
@@ -19,9 +19,105 @@ class ValidateSpecification extends Specification {
...
         when:
-        schema.validate(new JSONObject(new File("./invalid_example.json").text)) // throws a ValidationException if this object is invalid
+        schema.validate(new JSONObject(new File("./invalid_qbicId.json").text)) // throws a ValidationException if this object is invalid
```
Comment (sven1103) [inline validate.groovy]: rename file to `invalid_sampleId.json`?

File: validate.groovy [hunk 4]
```diff
@@ -19,9 +19,105 @@ class ValidateSpecification extends Specification {
...
+
+        then:
+        org.everit.json.schema.ValidationException exception = thrown(org.everit.json.schema.ValidationException)
+    }
+
+    def 'seqFacilityId exception'() {
```
Comment (sven1103) [inline validate.groovy]: More expressive description of the test method please, for all of them :)

**Review by sven1103 — CHANGES_REQUESTED** [review]

**Review by sven1103 — COMMENTED** [review]
👍

File: model.schema.json (hunk 1 above)
Comment (jenniferboedker) [reply model.schema.json]: Corrected with commit 5a14140

File: validate.groovy (hunk 2 above)
Comment (jenniferboedker) [reply validate.groovy]: solved by commit b297383

File: validate.groovy (hunk 3 above)
Comment (jenniferboedker) [reply validate.groovy]: solved by b297383

File: validate.groovy (hunk 4 above)
Comment (jenniferboedker) [reply validate.groovy]: solved by b297383

File: model.schema.json [hunk 5]
```diff
@@ -5,10 +5,62 @@
     "description": "A MetaStats object representation for statistical analysis",
     "type": "object",
     "properties": {
-        "qbicId": {
-            "description": "QBiC internal identifier for sample entities",
+        "prepSampleId": {
```
Comment (sven1103) [inline model.schema.json]:
```suggestion
        "samplePreparationId": {
```

File: model.schema.json [hunk 6]
```diff
@@ -5,10 +5,62 @@
...
+            "description": "QBiC internal identifier for preparation samples",
             "type": "string",
             "pattern": "Q[A-X0-9]{4}[0-9]{3}[A-X0-9]{2}"
+        },
+        "seqFacilityId": {
```
Comment (sven1103) [inline model.schema.json]:
```suggestion
        "sequencingFacilityId": {
```

File: model.schema.json [hunk 7]
```diff
@@ -5,10 +5,62 @@
...
+        },
+        "condition": {
+            "description": "Sample group characteristic",
+            "type": "string"
+        },
+        "fileName": {
+            "description": "Name of the data files for the sample",
+            "type": "string"
+        },
+        "seqDevice": {
```
Comment (sven1103) [inline model.schema.json]:
```suggestion
        "sequencingDevice": {
```

**Review by sven1103 — CHANGES_REQUESTED** [review]
Small changes left :)

File: model.schema.json (hunk 7 above)
Comment (jenniferboedker) [reply model.schema.json]: solved with c1e0309

File: model.schema.json (hunk 6 above)
Comment (jenniferboedker) [reply model.schema.json]: solved with c1e0309

File: model.schema.json (hunk 5 above)
Comment (jenniferboedker) [reply model.schema.json]: solved with c1e0309

**Review by sven1103 — APPROVED** [review]
🎉 good job

**Pull request merged**